charts/hlf-k8s/README.md

```diff
@@ -240,6 +241,21 @@ monitor:
 
 Check the pod's logs to get the list of organizations currently present in each channel.
 
+### Discover monitor pod
+
+The discover monitor pod periodically displays the list of organizations that are connected to the application channel with the hlf [service discovery](https://hyperledger-fabric.readthedocs.io/en/release-1.4/discovery-cli.html).
```

Comment:
Could you explain the difference with the monitor pod (joined v.s. connected), and in which case this is useful?

Reply:
Monitor does not show joined peer but added. I fix it.